pkg/trace/call_option.go

```diff
@@ -69,6 +69,17 @@ func WithCallOptions(ctx context.Context, opts CallOptions) {
 //	ctx = trace.StartCall(ctx, "http", trace.WithInfoLoggingDisabled())
 func WithInfoLoggingDisabled() call.Option {
 	return func(c *call.Info) {
-		c.Opts.DisableInfoLogging = true
+		c.Opts.EnableInfoLogging = false
+	}
+}
+
+// WithInfoLoggingEnabled enables info logging on the current call
```

Comment:
Maybe note here that based on the global value, this may have no effect?

Reply:
Good point, will document.